rhea/system/memmap/memmap.py

```diff
@@ -228,18 +228,18 @@ def map_from_generic(self, generic):
         bus.  This is a module and returns myhdl generators
 
         Arguments
-        ---------
+
         generic: The generic memory-mapped bus, all the memory-mapped
         supported modules use the *generic* bus internally.  This provides
         an agnostic bus interface to all the modules.
 
         Returns
-        -------
+
         myhdl generators
         """
         raise NotImplementedError
 
-    def peripheral_regfile(self, regfile, name, base_address=0):
+    def peripheral_regfile(self, glbl, regfile, name, base_address=0):
```

Comment:
The `glbl` should not be needed as an argument because `glbl` is already passed to constructor.  Was it is incorrectly somewhere?